src/basics.php

```diff
@@ -120,6 +121,20 @@ function breakpoint(): ?string
     }
 }
 
+if (!function_exists('table') && (PHP_SAPI === 'cli' || PHP_SAPI === 'phpdbg')) {
+    /**
+     * REPL convenience function to load a table
+     *
+     * @param string $alias The alias name you want to get.
+     * @param array $options The options you want to build the table with.
+     * @return \Cake\ORM\Table
+     */
+    function table(string $alias, array $options = []): Table
+    {
+        return Cake\ORM\TableRegistry::getTableLocator()->get($alias, $options);
```

Comment:
You can add `use` statement and just use class name here.

Reply:
Fixed